foreman/data_refinery_foreman/surveyor/test_external_source.py

```diff
@@ -25,9 +25,19 @@ def test_queue_downloader_jobs(self, mock_send_task):
 
         sample_object_1 = Sample()
         sample_object_1.accession_code = "Sample1"
+        sample_object_1.platform_accession_code = "Illumina Genome Analyzer"
+        sample_object_1.platform_accession_name = "Illumina Genome Analyzer"
```

Comment:
Duplicated line

Reply:
Not quite, one is setting the code the other is setting the name.